.github/workflows/main.yml

```
@@ -2,173 +2,260 @@ name: docker-wikibase-actions-workflow
-        - name: Build and push Docker jobrunner image
-          uses: docker/build-push-action@v5.3.0
-          with:
-            context: backup
-            push: ${{ env.PUSH_IMAGE }}
-            tags: ${{ steps.meta.outputs.tags }}
-            labels: ${{ steps.meta.outputs.labels }}
-            cache-from: type=gha
-            cache-to: type=gha,mode=max
+            BASE_IMAGE=${{ needs.build-staging.outputs.single-tag }}
+          cache-from: type=gha,scope=dev
+          cache-to: type=gha,mode=max,scope=dev
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)